src/Core/src/Handlers/View/ViewHandlerOfT.Windows.cs

```diff
@@ -36,7 +36,10 @@ protected override void SetupContainer()
 		protected override void RemoveContainer()
 		{
 			if (PlatformView == null || ContainerView == null || PlatformView.Parent != ContainerView)
+			{
+				ContainerView = null;
```

Comment:
From what I can tell, our tests that verify if `Clip` adds a `Container` haven't been running correctly on Windows.

When ran on windows `RemoveContainer` would run after `SetupContainer` but the `ContainerView` would remain set because we weren't nulling it out here.